src/components/molecules/VenueRoomItem/VenueRoomItem.scss

```diff
@@ -0,0 +1,22 @@
+@import "scss/constants.scss";
+
+.VenueRoomItem {
+  padding: $spacing--md;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  &:hover {
+    background-color: $saturated-black;
+  }
+
+  &__room-icon {
+    height: 34px;
+    width: 34px;
+    margin-right: 10px;
+    background-size: contain;
+    background-position: center;
+    border-radius: 4px;
```

Comment:
can you please add border-radius-xs?
![image](https://user-images.githubusercontent.com/63194656/126463624-6fcc2b40-408c-48eb-a4b1-fa111e6f23fc.png)

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1583/commits/f8fa0c67c34ac626919737b4bbd112b4bd715553